notebooks/official/training/hyperparameter_tuning_xgboost.ipynb

```diff
@@ -925,7 +840,7 @@
       "source": [
         "### Display the hyperparameter tuning job trial results\n",
         "\n",
-        "After the hyperparameter tuning job has completed, the property `trials` will return the results for each trial."
+        "Once the hyperparameter tuning job gets finished successfully, you can access the results from each trial using the `trials` property."
```

Comment:
change to: 
- "Once the hyperparameter tuning job successfully finishes, you can...